nawab_bot.py

```diff
@@ -150,9 +153,26 @@ def nawab_retweet_tweet(api):
 
 
 def main():
-   api = nawab_twitter_authenticate()
-   nawab_curate_list(api)
-   nawab_retweet_tweet(api)
+    
+    # default directory
+    def_dir = 'var/log'
+    if not os.path.exists(def_dir):
+        os.makedirs(def_dir)
+        os.chdir(def_dir)
+    # initialise the parser
+    parser = argparse.ArgumentParser(
+        description="Provide  path for logs destination"
+    )
+    # add argument to provide path 
+    parser.add_argument('--path', type=str 
+                       , help= ' path to destination directory')
+    args = vars(parser.parse_args())
+    ## To check if new_path is passed
+    if(args['path']):
+        os.chdir(args['path'])
```

Comment:
You are changing the directory, which makes no sense to me. Can you explain why are you doing that?